src/storage.js

```diff
@@ -198,14 +198,26 @@ export class LocalStorageStore {
     }
   }
 
-  cleanWallet() {
-    this.removeItem(IS_HARDWARE_KEY);
-    this.removeItem(CLOSED_KEY);
+  /**
+   * Clean access data from persistent storage, making the wallet 'not loaded'
+   */
+  cleanAccessData() {
```

Comment:
Why did you need to change this?

Reply:
When starting a hardware wallet we were having an issue that the acess data was being deleted during the `startWallet` saga, this would fail the process to start a wallet and if we clicked on retry it would work.

The issue was that we were being redirected to the locked screen as a hardware wallet, and we call `cleanWallet` to show the `wallet_type` screen.
This was fixed by not redirecting the hardware wallet to the locked screen during the `startWallet` saga and not cleaning the access data on `cleanWallet`, this is why I created the `cleanAccessData` if we really want to delete the access data from the localStorage.